core/math/vector2.cpp

```diff
@@ -171,6 +171,13 @@ Vector2 Vector2::move_toward(const Vector2 &p_to, real_t p_delta) const {
 	return len <= p_delta || len < (real_t)CMP_EPSILON ? p_to : v + vd / len * p_delta;
 }
 
+Vector2 Vector2::move_toward_smooth(const Vector2 &p_to, real_t p_delta, real_t p_rate) const {
+	real_t amount = -expm1(-p_delta * p_rate);
+	return Vector2(
+			Math::lerp(x, p_to.x, amount),
+			Math::lerp(y, p_to.y, amount));
```

Comment:
```suggestion
	return lerp(p_to, -expm1(-p_delta * p_rate));
```
Avoid duplicating code, and clearer IMO

Reply:
Oh I couldn't find the method the first time I looked it up for some reason. Yeah that's much better, changed.